includes/transformer/class-post.php

```diff
@@ -642,9 +642,15 @@ protected function get_type() {
 			return \ucfirst( $post_format_setting );
 		}
 
-		$has_title = post_type_supports( $this->wp_object->post_type, 'title' );
-
-		if ( ! $has_title ) {
+		$has_title = \post_type_supports( $this->wp_object->post_type, 'title' );
```

Comment:
This will only check if the post_type supports having a title, not if there is one. Probably want something more like this?

```suggestion
		$has_title = \post_type_supports( $this->wp_object->post_type, 'title' ) && ! empty( get_the_title( $this->wp_object );
```

Reply:
I check for a title some lines below (line 651).

Reply:
Ah! Should have kept reading 😅